docs/virtual_machines/lifecycle.md

```diff
@@ -45,6 +45,17 @@ corresponding `VirtualMachineInstance` object using `kubectl`.
 > deleted from the cluster. You will not be able to start this
 > VirtualMachineInstance object again.
 
+## Starting and stopping a virtual machine
+
+Virtual machines, in contrast to VirtualMachineInstances, have a running state. Thus on VM you can define if it
+should be running, or not. VirtualMachineInstaces are, if they are defined in the cluster, always running and consuming resources.
+
+`virtctl` is used in order to start and stop a VirtualMachine:
+
+    $ virtctl start my-vm
+    $ virtctl stop my-vm
+    # Or force stop a VM by
+    $ virtctl stop my-vm --grace-period 0 --force
```

Comment:
Perhaps, we should add a note: `Note: Using `--force` immediately removes the VirtualMachineInstance pod from API and bypasses graceful deletion. This can result in inconsistency or data loss.`

Reply:
@fossedihelm see line 16 - there we create a VM

Fixed the wording